src/app/modules/main/profile_section/view.nim

```diff
@@ -1,15 +1,38 @@
 import NimQml
 
+import ./io_interface
+
 QtObject:
   type
     View* = ref object of QObject
+      delegate: io_interface.AccessInterface
+      # TODO: move to the correct module once all have been merged
```

Comment:
Is this TODO valid? what should the correct module be?

Reply:
yes it is, and I don´t know which module it should be as there is no advance section module so far.